src/bfcli/parser.y

```diff
@@ -315,6 +319,18 @@ rule_option     : LOG LOG_HEADERS
                         .flags = BF_RULE_OPTION_COUNTER,
                     };
                 }
+                | RATELIMIT RATELIMIT_VAL
+                {
+                    _cleanup_free_ char *in = $2;
+                    char *tmp = in;
+                    uint32_t limit = atoi(tmp);
```

Comment:
`strtoul` with overflow check would be better, see https://github.com/facebook/bpfilter/blob/main/src/libbpfilter/matcher.c#L283.